scene/2d/area_2d.cpp

```diff
@@ -157,8 +163,14 @@ void Area2D::_body_exit_tree(ObjectID p_id) {
 	ERR_FAIL_COND(!E->value.in_tree);
 	E->value.in_tree = false;
 	emit_signal(SceneStringNames::get_singleton()->body_exited, node);
+	if (node && node->has_signal(SceneStringNames::get_singleton()->exited_area)) {
```

Comment:
```suggestion
	if (node->has_signal(SceneStringNames::get_singleton()->exited_area)) {
```
In the cases where no `if (node)` checks were added before this is unnecessary